nft-rfc-006.md

```diff
@@ -0,0 +1,860 @@
+observers could not directly tell if those service endpoints mean that
+the Twitter account controls those DIDs, they do gain a point of
+observation for consideration in evaluating that correlation. For many,
+this undermines the privacy goals of being able to use DIDs without
+revealing any additional personally identifiable information.
+
+Third, DIDs, and their sister specification VCs, embrace an open world
+data model, in which anyone can add any parameter to the Document
+without violating conformance. This is convenient, but also means that
+unfortunate properties can and will be used, such as properties that
+explicitly communicate details about the DID Subject. Such details are
+intended to be preserved by Methods and requesting parties even when
+they are misunderstood, which creates significant opportunity for
+unintended consequences. However, it is understood that this flexibility
+affords significant innovation even as it invites endless debate on
+which properties get to be defined where---a decision that has
+significant impact on ubiquitous adoption.
+
+Finally, DIDs are NOT signed documents. The only verification afforded
+by DIDs is that by executing the process defined in the DID Method,
+whatever the result is \*is\* the verified DID Document. This means that
+you MUST trust the software that performs this resolution, either by
+reputation or because you wrote it yourself. This choice was informed by
+a tricky catch-22 where DID identifiers are a result of cryptographic
+operations perform either on or with the DID Document. For example, IPFS
+based DID Documents cannot contain the IPFS address because the content
+hash of that DID Document is the address. Similarly, ledger-based DIDs
+that depend on blockheight determine their ID through the process of
+registration. In both cases, you can\'t fully create the DID Document
+before it is registered. I may be possible to ensure a signature in the
+DID resolver metadata as a signed hash using the cryptographic material
+in that document, but this remains an area of innovation.
+
+**A note on DIDs as URIs rather than IRIs**
+
+DIDs and IIDs are not intended to be human readable, so the spoofing
+concern is mitigated to some degree as is the need to support
+alternative character sets. In fact, DIDs and IIDs are generally
+encodings of cryptographic identifiers, such as a public key; none of
+the extant encodings use non-ascii characters.
+
+Proposal 
+---------
+
+**First**, create IIDs as an interoperable family DIDs with additional
+properties. Each IID \"namespace\"---to use the CAIP term---defines its
+own Interchain compatible DID method, e.g., did:erc721, did:cosmos,
+did:iov, did:etc. DID Methods that support IID functionality define
+themselves as an IID Method. IID semantics, shared among different IID
+Methods, enable cross-chain interoperability with other IIDs for NFT and
+blockchain functionality.
+
+In short, IIDs **ARE** DIDs.
```

Comment:
`In short, IIDs **ARE** DIDs.` => maybe this is alluding to the fact that IIDs and DIDs are interoperable, but if someone googles IID, the W3C spec will not be found, so it adds ambiguity to the definition which is something we should try to avoid. Any thoughts here?

https://www.google.com/search?q=IIDs+interchain+identifiers

Reply:
Well, only this repo identifies IIDs, so I wouldn't expect Google to find it.

I don't mean just that IIDs are interoperable with DIDs, but that IIDs are literally DIDs. All IID Methods should be listed in the DID Spec Registry, ideally with something like "An IID Method" in the brief description so people can easily discern the IIDs among the other methods.

Again, this is similar to the Sidetree family of DIDs.

Reply:
Ok, this makes more sense, thanks for explaining :handshake: